src/systemTest/java/com/flightstats/hub/model/ChannelItemPathParts.java

```diff
@@ -0,0 +1,24 @@
+package com.flightstats.hub.model;
+
+import lombok.Builder;
+import lombok.Getter;
+import org.joda.time.DateTime;
+
+@Getter
```

Comment:
Because this isn't technically a new class; it's just refactored/renamed and I forgot to change it. I will do so.

Reply:
Oops, sorry, I didn't mean to flag a renamed class! I usually try to skip those in review, but it can be  hard to tell (well, for me, not familiar with the code base) what's moved vs what's new.

Reply:
No worries. I generally do try to clean up stuff I see when I move stuff around, so I appreciate the comments.